Wikipedia/Code/ArticleViewController+ArticleInformation.swift

```diff
@@ -56,14 +56,15 @@ extension ArticleViewController {
             return
         }
         let issues = payload.compactMap { ($0["html"] as? String)?.removingHTML }
+        let splitSeperator = Character("\n")
+        let multipleIssuesMessageArray = issues[0].split(separator: splitSeperator)
```

Comment:
Nice - you can draw the formatting lines out into their own method to tidy things up a little bit more too.